docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md

```diff
@@ -190,39 +190,39 @@ You are now ready to create your first **Collection Type**.
 
 A **Content Type** can be considered a sort of _blueprint_ for the data created. In other words, a **Content Type** is the schema of the data structure.
 
-Both **Collection Type** and **Single Type** are types of **Content Type**. **Collection Type** is used for content that repeats such as restaurants, or blog posts (hence "collection"). **Single Type** is used to manage content that will have only one instance - for example, your menu, a static page, or a footer.
+Both **Collection Type** and **Single Type** are types of **Collection Type**. **Collection Type** is used for content that repeats such as restaurants, or blog posts (hence "collection"). **Single Type** is used to manage content that will have only one instance - for example, your menu, a static page, or a footer.
```

Comment:
Here it should be `Content Type`

Both **Collection Type** and **Single Type** are types of **Content Type**